### ISSUE TITLE
Insta-kill character voicelines not playing

### ISSUE THREAD

**Issue by Anchovio123** [Jbleezy/BO1-Reimagined#29]
I do not know if this was intentional or not, and this is a very minor issue, but the VOX that usually plays when you knife a zombie on insta-kill seems to be absent. Nothing to report other than that, and thank you JBleezy for making such a great mod (and making Ascension's monkeys not unbearable)!

**Comment by Jbleezy** [comment]
Do you have any more details? Do you mean the announcer vox or the character dialog?

**Comment by Anchovio123** [comment]
No, the announcer is working fine, I meant the character dialogue doesn’t play.
